packages/renderer/src/App.svelte

```diff
@@ -178,6 +178,12 @@ window.events?.receive('kubernetes-navigation', (args: unknown) => {
         <Route path="/images/existing-image-create-container" breadcrumb="Select image" >
           <CreateContainerFromExistingImage />
         </Route>
+        <Route path="/images/build" breadcrumb="Build an Image">
+          <BuildImageFromContainerfile />
+        </Route>
+        <Route path="/operations/images/build/:taskId" breadcrumb="Build an Image" let:meta>
+          <BuildImageFromContainerfile taskId={+meta.params.taskId}/>
```

Comment:
question: not sure to understand the `+` prefixing the `meta.params.taskId`?

Reply:
It converts string to number.

https://stackoverflow.com/questions/12862624/whats-the-fastest-way-to-convert-string-to-number-in-javascript